src/views/Settings.vue

```diff
@@ -55,6 +55,22 @@
             </v-card-text>
           </v-card>
 
+          <v-card class="elevation-12 mt-5">
+            <v-toolbar dark>
+              <v-toolbar-title class="white--text">Theme</v-toolbar-title>
+            </v-toolbar>
+            <v-card-text>
+              <div class="body-1 mb-3">
+                Select light or dark mode.
+                <v-switch
+                  color="primary"
+                  v-model="localDarkMode"
```

Comment:
bind to `darkMode` and remove ALL the `localDarkMode`declaration and references. The `darkMode` getter and setter should do the trick